### PR TITLE
Add legacy statistics log type

### PR DESCRIPTION
**Pull request by mhilton**
Add an additional log type for legacy statistics.

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results: http://ci.jujugui.org:8080/job/charmstore/647/

File: params/params.go [hunk 1]
```diff
@@ -220,4 +220,7 @@ const (
 // responses.
 type LogType string
 
-const IngestionType LogType = "ingestion"
+const (
+	IngestionType        LogType = "ingestion"
+	LegacyStatisticsType         = "legacyStatistics"
```
Comment (rogpeppe) [inline params/params.go]: need LogType type here.

File: internal/v4/log_test.go [hunk 2]
```diff
@@ -114,13 +121,14 @@ var getLogsTests = []struct {
 	about:       "use limit",
 	querystring: "?limit=2",
 	expectBody: []*params.LogResponse{
+		logResponses["stats"],
 		logResponses["error3"],
-		logResponses["info3"],
```
Comment (kadams54) [inline internal/v4/log_test.go:118]: Out of curiosity, is this removal (and the one on line 147) a drive-by cleanup?

**Comment by rogpeppe** [comment]
LGTM. No QA.

File: params/params.go (hunk 1 above)
Comment (mhilton) [inline params/params.go]: do I? I though const blocks retained the information from the line above if it isn't specified.

File: internal/v4/log_test.go (hunk 2 above)
Comment (mhilton) [inline internal/v4/log_test.go:118]: because the limit is 2 and my new one at the top pushed out the one at the bottom.

**Comment by kadams54** [comment]
:+1: No QA.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results: http://ci.jujugui.org:8080/job/charmstore/650/

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results: http://ci.jujugui.org:8080/job/charmstore/652/

**Comment by mhilton** [comment]
I'm going to :shipit: as this is fairly trivial and needed for subsequent developments.

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/charmstore-merge
